strings/base_coroutine_threadpool.h

```diff
@@ -50,6 +50,23 @@ namespace winrt::impl
 
     struct resume_apartment_context
     {
+        resume_apartment_context() = default;
+        resume_apartment_context(std::nullptr_t) : m_context(nullptr), m_context_type(-1) {}
+        resume_apartment_context(resume_apartment_context const&) = default;
+        resume_apartment_context(resume_apartment_context&& other) noexcept :
+            m_context(std::move(other.m_context)), m_context_type(std::exchange(other.m_context_type, -1)) {}
+        resume_apartment_context& operator=(resume_apartment_context const&) = default;
+        resume_apartment_context& operator=(resume_apartment_context&& other) noexcept
+        {
+            m_context = std::move(other.m_context);
+            m_context_type = std::exchange(other.m_context_type, -1);
+            return *this;
+        }
+        bool valid() const
```

Comment:
noexcept

Reply:
Thanks. I've been wary of doing too much noexcept after being bitten by VS2017's inability to `noexcept = default`. Need to overcome my shyness.

Reply:
The landmines are everywhere.